web/src/components/NavigationBarScrollContainer.tsx

```diff
@@ -0,0 +1,106 @@
+import React, { ReactElement, ReactNode, RefObject, useEffect, useState } from 'react'
+import styled, { css } from 'styled-components'
+
+import { UiDirectionType } from 'translations/src'
+
+import iconArrowBack from '../assets/IconArrowBack.svg'
+import iconArrowForward from '../assets/IconArrowForward.svg'
+import dimensions from '../constants/dimensions'
+
+type PropsType = {
+  children: ReactNode
+  direction?: UiDirectionType
+  scrollContainer: RefObject<HTMLDivElement>
+}
+
+const Container = styled.div`
+  display: flex;
+`
+
+const Arrow = styled.img<{ direction?: string; visible: boolean }>`
+  width: 12px;
+  height: 10px;
+  padding: 0 8px;
+  align-self: center;
+  opacity: 0;
+
+  ${props =>
+    props.direction === 'rtl' &&
+    css`
```

Comment:
Same, why do we need these helpers? Again belowl.

Reply:
removed helpers